controllers/tempo/tempostack_controller.go

```diff
@@ -139,7 +139,7 @@ func (r *TempoStackReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 //
 //   - For any other error: Set the status condition to Failed,
 //     the Reason to "FailedReconciliation" and the message to the error message.
-func (r *TempoStackReconciler) handleReconcileStatus(ctx context.Context, log logr.Logger, tempo v1alpha1.TempoStack, reconcileError error) (ctrl.Result, error) {
```

Comment:
@iblancasa @andreasgerstmayr I a comment here to ignore this rule, but might need to change something here?

Reply:
I think ignoring the linter rule is fine, it's just complaining that the first return parameter is always `ctrl.Result{}`